### ISSUE TITLE
Give your good suggestions

### ISSUE THREAD

**Issue by joansnotion** [joansnotion/NotionMate#9]
Leave your suggestion, if it is a good one, I will take it and add it in a later version. Thank you!

**Comment by glen-handsaw-b4c** [comment]
Hello! I think the system used in gitbook to go to the next/previous page is pretty good.
![image](https://github.com/joansnotion/NotionMate/assets/170483937/7d7c022a-5759-4543-810f-314002db6478)

**Comment by glen-handsaw-b4c** [comment]
Hello! I would really appreciate if you create something like a version control to see what is new in each update! Thanks!

**Comment by joansnotion** [comment]
This is great suggestions!

**Comment by glen-handsaw-b4c** [comment]
When you start to work with Code syntax highlighting, take a look at long codes (more than 250 lines) which don't have syntax highlighting by notion itself, it creates some bugs with the extension Notion Syntax Highlighter

**Comment by joansnotion** [comment]
Do you mean to let Motion Mate support code syntax highlighting?

**Comment by glen-handsaw-b4c** [comment]
Yes! Besides offering themes, change the Notion's Code Syntax (which is terrible) like is done in this [extension](https://chromewebstore.google.com/detail/notion-syntax-highlighter/omapdeaklaaabcegmdikkciplbgignak)

**Comment by joansnotion** [comment]
Because this feature is dependent on third-party libraries, it affects notion's performance, so I prefer to stand alone with this feature rather than integrate it into Notion Mate.

**Comment by studentPackpro** [comment]
Hello! Could make the left side bar wrap the titles, they get the ... and I cannot read the titles when there are more than 2 levels. Thank you!

**Comment by joansnotion** [comment]
> Hello! Could make the left side bar wrap the titles, they get the ... and I cannot read the titles when there are more than 2 levels. Thank you!

Can you show your idea through a screenshot?

**Comment by studentPackpro** [comment]
![2024-10-22_14-26](https://github.com/user-attachments/assets/9a8ab937-a19d-495b-b2f4-60fa3da5222e)
It would be something like that.

**Comment by joansnotion** [comment]
Thanks for the suggestion, I will consider whether to add it or not, thanks for your support!

**Comment by scplay** [comment]
It would be great to toggle back to no sequential number in, when I accidently click this "H" in TOC and switch to another page it's impossible to simply revert using Crtl + Z

<img width="346" alt="Image" src="https://github.com/user-attachments/assets/c02bd4f1-46b2-456a-8ee5-df006e30edd3" />

Or. what I mean is just trying to match ^\d+\. to delete all when click it again

**Comment by joansnotion** [comment]
Because switching to a new page, there is no way to recover the last record, you can try to recover it through the page history.

**Comment by scplay** [comment]
As the menu grows too long

<img width="351" alt="Image" src="https://github.com/user-attachments/assets/55acf12c-862f-4787-b7fc-1fc634dcab55" />

Would you like to provide a search input for convenience like NotionBoost

<img width="552" alt="Image" src="https://github.com/user-attachments/assets/2ef94f6e-3b00-42f7-9abb-f4f71cc79ade" />

And it would be great to provide a image preview of the result as there are some features when I toggle it On and Off, I can't find any difference

<img width="345" alt="Image" src="https://github.com/user-attachments/assets/e4b409aa-789e-47b1-b2c4-e52452f62746" />

**Comment by joansnotion** [comment]
Thanks for the suggestion, we'll add an input search function, and feature previews can be viewed at notionmate.top!

**Comment by scplay** [comment]
Another good feature from NotionBoost is it can automatically left align all images when set image left align feature On, I think it would be better to add this to a switcher, not a command which I need an extra click in every page. 

<img width="214" alt="Image" src="https://github.com/user-attachments/assets/19ca64db-4718-4623-8985-27b52cc7a5a1" />

![Image](https://github.com/user-attachments/assets/580f1b2b-822f-4faa-9ad8-36ed68e5b9a3)

Thank you for your time.

**Comment by joansnotion** [comment]
![Image](https://github.com/user-attachments/assets/5e5e6d62-ac31-441e-a486-0e23ee774298)

![Image](https://github.com/user-attachments/assets/a731568c-4878-4637-9c71-c834cfe49895)

Currently notionmate supports such a feature as shown above.

**Comment by scplay** [comment]
> ![Image](https://github.com/user-attachments/assets/5e5e6d62-ac31-441e-a486-0e23ee774298)
> 
> ![Image](https://github.com/user-attachments/assets/a731568c-4878-4637-9c71-c834cfe49895)
> 
> Currently notionmate supports such a feature as shown above.

Appricate your work, I think I need to make it more clearly, I know the feature above, But it need an extra click when I upload any new image or switch to another page, however the NotionBoost's left align can automatically apply to every page or new image uploaded

**Comment by joansnotion** [comment]
You are right, the next version will add an option for automatic application to pages.

**Comment by joansnotion** [comment]
> As the menu grows too long
> 
> <img alt="Image" width="351" src="https://private-user-images.githubusercontent.com/7413214/428373994-55acf12c-862f-4787-b7fc-1fc634dcab55.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDMzNTAyMTMsIm5iZiI6MTc0MzM0OTkxMywicGF0aCI6Ii83NDEzMjE0LzQyODM3Mzk5NC01NWFjZjEyYy04NjJmLTQ3ODctYjdmYy0xZmM2MzRkY2FiNTUucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDMzMCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTAzMzBUMTU1MTUzWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9M2RjZTY0NWE5NTI3NTI1MWU5YzUyMmUwZjA1NjdlMWFkMmJhMTdmMGEyZWY0N2FjNDhiM2MxYjRhMTgwNWQ0MSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.9rd1F7nl67vc-fFSzyLoopgWE7BNey0j9AaNdDm32AQ">
> Would you like to provide a search input for convenience like NotionBoost
> 
> <img alt="Image" width="552" src="https://private-user-images.githubusercontent.com/7413214/428373984-2ef94f6e-3b00-42f7-9abb-f4f71cc79ade.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDMzNTAyMTMsIm5iZiI6MTc0MzM0OTkxMywicGF0aCI6Ii83NDEzMjE0LzQyODM3Mzk4NC0yZWY5NGY2ZS0zYjAwLTQyZjctOWFiYi1mNGY3MWNjNzlhZGUucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDMzMCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTAzMzBUMTU1MTUzWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZmU5MzFmNzkyZDBjYjgzNDAyMDM1NjQ0NmEwYzNiMmM4ODY5N2MwZTk1ODU0NmM1NGRkMzJiNDRkMjhmNGE1OCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.6LvPDl7AKvMQofg5lYSjv_ukQwNyBxMFaxKeO79Tw9Y">
> And it would be great to provide a image preview of the result as there are some features when I toggle it On and Off, I can't find any difference
> 
> <img alt="Image" width="345" src="https://private-user-images.githubusercontent.com/7413214/428374062-e4b409aa-789e-47b1-b2c4-e52452f62746.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDMzNTAyMTMsIm5iZiI6MTc0MzM0OTkxMywicGF0aCI6Ii83NDEzMjE0LzQyODM3NDA2Mi1lNGI0MDlhYS03ODllLTQ3YjEtYjJjNC1lNTI0NTJmNjI3NDYucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDMzMCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTAzMzBUMTU1MTUzWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9OTViZDZmMzAxNWQ0YjJkZjJkNTNiYTQ2ZDc5ZTA3MDJiZTRlZTcyYmEwMTU0MmFlMjM0ZjA1OWNiYmQwMmIzOSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.cU_jlFcN9Rp94rqZgf2kkoamk9hyLBPqDLaxoMtE4y0">

Please wait for notionmate version 2.3.7. 
The new version adds a feature input search box to align media file features.

![Image](https://github.com/user-attachments/assets/6dd5f85d-69f7-42be-8a61-55b378a626b7)

**Comment by bencwills** [comment]
Pardon my ignorance but is there any way the plugin can add some sort of block tagging functionality to Notion? You can tag in databases and @mention in blocks, but what I really want to do is have a simple way of seeing every block with a particular tag in a new page or DB. Thanks!

I was curious if that's what "header hashtag symbols" was but in the documentation you have your website it's missing.

<img width="964" height="715" alt="Image" src="https://github.com/user-attachments/assets/4b52c42f-645f-468a-9a47-fb059e9d67a2" />

**Comment by bencwills** [comment]
Also—would love a feature that allows me to expand or collapse all toggle blocks at once!!! You can do this in Word with Styles (R click → Headings→collapse heading or something like that). 

Thanks!

**Comment by joansnotion** [comment]
Thanks for the suggestion, notion supports collapsing or expanding headers, which can be done via toggle block

**Comment by bencwills** [comment]
> Thanks for the suggestion, notion supports collapsing or expanding headers, which can be done via toggle block

Yes! Toggle Headings are my most used block after text. I was looking for a shortcut to toggle them open. For anyone who comes across this thread, it is Ctrl+Alt+T to toggle them all open or closed. What I would love is default Toggles Open setting.

**Comment by joansnotion** [comment]
> > Thanks for the suggestion, notion supports collapsing or expanding headers, which can be done via toggle block
> 
> Yes! Toggle Headings are my most used block after text. I was looking for a shortcut to toggle them open. For anyone who comes across this thread, it is Ctrl+Alt+T to toggle them all open or closed. What I would love is default Toggles Open setting.

<img width="600" height="792" alt="Image" src="https://github.com/user-attachments/assets/96505842-f97c-432e-8ab4-6ba4b869adef" />

Hi, the latest version v2.4.0 already supports expanding or collapsing toggle heading, thanks for your suggestion!

**Comment by tokiyaaki-max** [comment]
Hello!!
Love the work by the way!!

A few suggestions:
- [x] Are we able to make it a setting so that bullet points, toggle lists, and numbered lists also have narrow row spacing? **Narrow Row Spacing** only works for to-do lists and Headers, as far as I can tell.
- [ ] Alternatively, if someone prefers to have some padding between the options (to-do list, headers, bullet points, numbered lists and toggle lists), could implement a setting option to allow the user to customize the spacing - like 3px height, 4px height, etc. (not sure what default is)
- [x] For the **Change Custom Font**, it only modifies H1 and not H2 or H3. Is it possible to make it affect all? Or allow a different font for each?

<img width="252" height="458" alt="Image" src="https://github.com/user-attachments/assets/32b5b217-27f4-4eeb-b17d-1219626cbdf0" />

Thank you for the amazing work!!

**Comment by bencwills** [comment]
> Hello!! Love the work by the way!!
> 
> A few suggestions:
> 
> * [ ]  Are we able to make it a setting so that bullet points, toggle lists, and numbered lists also have narrow row spacing? **Narrow Row Spacing** only works for to-do lists and Headers, as far as I can tell.[ ]  Alternatively, if someone prefers to have some padding between the options (to-do list, headers, bullet points, numbered lists and toggle lists), could implement a setting option to allow the user to customize the spacing - like 3px height, 4px height, etc. (not sure what default is)

Agree! I mostly use bullets and the narrow row spacing isn't narrow enough.

**Comment by joansnotion** [comment]
Thank you very much, I will you guys suggestions, please wait for the next version!

**Comment by PLA0185** [comment]
<img width="209" height="55" alt="Image" src="https://github.com/user-attachments/assets/51b3a108-f0c2-42ee-a725-a4c188157974" />
您能让表格属性的图标“Aa”取消显示吗

**Comment by PLA0185** [comment]
还有一个功能就是类似于以下几个帖子里所发的，在gallery视图的属性当中添加属性栏名称：
https://www.reddit.com/r/Notion/comments/s6e1xt/how_to_show_properties_titles_on_cards_in_notion/?tl=zh-hans
https://www.reddit.com/r/Notion/comments/zsvwh0/how_do_you_include_property_text_when_showing_it/?tl=zh-hans
https://www.reddit.com/r/Notion/comments/zxp5a0/does_anyone_know_how_to_make_the_properties/?show=original

**Comment by joansnotion** [comment]
> <img alt="Image" width="209" height="55" src="https://private-user-images.githubusercontent.com/129172748/495427272-51b3a108-f0c2-42ee-a725-a4c188157974.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTk2NjY1MjcsIm5iZiI6MTc1OTY2NjIyNywicGF0aCI6Ii8xMjkxNzI3NDgvNDk1NDI3MjcyLTUxYjNhMTA4LWYwYzItNDJlZS1hNzI1LWE0YzE4ODE1Nzk3NC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMDA1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTAwNVQxMjEwMjdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lYTFhZWU5YzM5YTBjMDUwZjc2NjY0M2YxNmUxNDJjNGY1ZTI5YmE4ZGUwMmI5ZWE5NGI2Y2I5MGMwMjk5NTdjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.LHxKDIhPTpyC-bRqeAd6oYIdQ11aa8CXoixPz1Rxsyo"> 您能让表格属性的图标“Aa”取消显示吗

Yes, but I don't think it's necessary.

**Comment by joansnotion** [comment]
> Hello!! Love the work by the way!!
> 
> A few suggestions:
> 
> * [x]  Are we able to make it a setting so that bullet points, toggle lists, and numbered lists also have narrow row spacing? **Narrow Row Spacing** only works for to-do lists and Headers, as far as I can tell.[ ]  Alternatively, if someone prefers to have some padding between the options (to-do list, headers, bullet points, numbered lists and toggle lists), could implement a setting option to allow the user to customize the spacing - like 3px height, 4px height, etc. (not sure what default is)[x]  For the **Change Custom Font**, it only modifies H1 and not H2 or H3. Is it possible to make it affect all? Or allow a different font for each?
> 
> <img alt="Image" width="252" height="458" src="https://private-user-images.githubusercontent.com/234289186/493605061-32b5b217-27f4-4eeb-b17d-1219626cbdf0.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTk2NjczMjQsIm5iZiI6MTc1OTY2NzAyNCwicGF0aCI6Ii8yMzQyODkxODYvNDkzNjA1MDYxLTMyYjViMjE3LTI3ZjQtNGVlYi1iMTdkLTEyMTk2MjZjYmRmMC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMDA1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTAwNVQxMjIzNDRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1mY2YzMDE0ZTM2Nzc3ZjY3MzM3YjFmNDVlN2JiODQyZmQ0MDA2OGM2YzE2NzgyZDYzOGJiZTM4YWMyMzM1YjFhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.gWTdNKh9BIUrZ_8rVO1aDVWbp57FPyJo0PPvkpFklrM">
> Thank you for the amazing work!!

1. for both numbered list and bulleted list have been fixed, currently it is not possible to customize the height. 2.
2. the problem of not being able to change the header font has been fixed.
Please wait for the chorme store review, the new version v2.4.2 has fixed these issues.

**Comment by 00-star** [comment]
After I click the hearders button, there are additional numbers before the header.

<img width="432" height="562" alt="Image" src="https://github.com/user-attachments/assets/933f2c9a-c8d0-4e79-88d9-6227d49828ff" /> 
<img width="326" height="535" alt="Image" src="https://github.com/user-attachments/assets/29252146-80a3-484c-968c-9d52902483b2" />

**Comment by joansnotion** [comment]
This will be fixed in the next release, thanks for the feedback!

**Comment by 00-star** [comment]
Would you like to provide colorful identation lines like Obsidian?

<img width="493" height="316" alt="Image" src="https://github.com/user-attachments/assets/20bba7a0-13af-4e2a-b95d-41658400decc" />

**Comment by joansnotion** [comment]
This will be fixed in the next version, thanks for your suggestion!

**Comment by 00-star** [comment]
The "fixed favicon" function does not work well in Chrome.

<img width="1100" height="334" alt="Image" src="https://github.com/user-attachments/assets/895b917c-58ed-4ba3-aa6a-09d8f921f926" />

**Comment by joansnotion** [comment]
Thanks for the feedback, it will be fixed in the next version!

**Comment by joansnotion** [comment]
> Would you like to provide colorful identation lines like Obsidian?
> 
> <img alt="Image" width="493" height="316" src="https://private-user-images.githubusercontent.com/65845263/523234758-20bba7a0-13af-4e2a-b95d-41658400decc.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Njk3ODIzNzUsIm5iZiI6MTc2OTc4MjA3NSwicGF0aCI6Ii82NTg0NTI2My81MjMyMzQ3NTgtMjBiYmE3YTAtMTNhZi00ZTJhLWI5NWQtNDE2NTg0MDBkZWNjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNjAxMzAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjYwMTMwVDE0MDc1NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTAyY2RkNDYzNGM2MWJiYzE3NTljNjY1ZWQzZGRlMWE1NDFiMzk3MmM2MzkxNGVkYTE2MTNhYjQxZGVmOWM4ZDgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.Jp0NhX_wuTXMsvCrJ_fH_qqrL9qxnVX3GhvnLbeSqxU">

<img width="1110" height="453" alt="Image" src="https://github.com/user-attachments/assets/a4f3d3dd-df38-44d5-8f59-150a53d2bb87" />

This feature is already supported. Thank you for your feedback! Please update to v2.4.3.

**Comment by joansnotion** [comment]
> After I click the hearders button, there are additional numbers before the header.
> 
> <img alt="Image" width="432" height="562" src="https://private-user-images.githubusercontent.com/65845263/520489096-933f2c9a-c8d0-4e79-88d9-6227d49828ff.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Njk3ODIzNzUsIm5iZiI6MTc2OTc4MjA3NSwicGF0aCI6Ii82NTg0NTI2My81MjA0ODkwOTYtOTMzZjJjOWEtYzhkMC00ZTc5LTg4ZDktNjIyN2Q0OTgyOGZmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNjAxMzAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjYwMTMwVDE0MDc1NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTI3ODYyODc4ZmEzOWVjNTU5ZmQzZTUxYjI5OWM4MTcyMGZlNjhhYTM4ZmUzNGVmYmQzMTc4OWFhNTIwNTQ5NDMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.sAxxslxDQADaw_vkP2TCTqVfIT3IAAYlf4Mu8c9UbQM"> <img alt="Image" width="326" height="535" src="https://private-user-images.githubusercontent.com/65845263/520489095-29252146-80a3-484c-968c-9d52902483b2.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Njk3ODIzNzUsIm5iZiI6MTc2OTc4MjA3NSwicGF0aCI6Ii82NTg0NTI2My81MjA0ODkwOTUtMjkyNTIxNDYtODBhMy00ODRjLTk2OGMtOWQ1MjkwMjQ4M2IyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNjAxMzAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjYwMTMwVDE0MDc1NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTc4OWU3NTU0MTZhYzhlYWFjZWI1YWY1MTM3MWEzNTRlNjc4ZmM5ZmZlMWZiZjQxYWZmMDc4YzUyYTEyMjA4ZWMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.p8EUvSxsS-8SOS6Z7rAAdz8S75xHYFFJTcqQVp0G-78">

This issue has been fixed. Thank you for your feedback! Please update to v2.4.3.

**Comment by guigui0114** [comment]
Could you add a new CSS theme matching Feishu’s highlight color scheme?

<img width="400" height="751" alt="Image" src="https://github.com/user-attachments/assets/f22b5a67-6b5b-49bc-a0f1-a031565fc64f" />

```
<span style="background-color: #F2F3F5">背景高亮为浅灰色的文字。</span>
<span style="background-color: #FBBFBC">背景高亮为浅红色的文字。</span>
<span style="background-color: #FEDDB6">背景高亮为浅橙色的文字。</span>
<span style="background-color: #FEF794">背景高亮为浅黄色的文字。</span>
<span style="background-color: #C3EEC0">背景高亮为浅绿色的文字。</span>
<span style="background-color: #CDDAFB">背景高亮为浅蓝色的文字。</span>
<span style="background-color: #DCC9FB">背景高亮为浅紫色的文字。</span>
<span style="background-color: #E5E6E9">背景高亮为中灰色的文字。</span>
<span style="background-color: #BBBFC4">背景高亮为深灰色的文字。</span>
<span style="background-color: #F76964">背景高亮为红色的文字。</span>
<span style="background-color: #FFA53D">背景高亮为橙色的文字。</span>
<span style="background-color: #FFE928">背景高亮为黄色的文字。</span>
<span style="background-color: #62D256">背景高亮为绿色的文字。</span>
<span style="background-color: #9EBBFE">背景高亮为蓝色的文字。</span>
<span style="background-color: #C4A4FA">背景高亮为紫色的文字。</span>

<span style="color: #D5D7DA;">这是灰色的字体。</span>
<span style="color: #DB4F48;">这是红色的字体。</span>
<span style="color: #E28922;">这是橙色的字体。</span>
<span style="color: #DDA011;">这是黄色的字体。</span>
<span style="color: #3DA731;">这是绿色的字体。</span>
<span style="color: #4372DE;">这是蓝色的字体。</span>
<span style="color: #6B31CB;">这是紫色的字体。</span>
```

**Comment by joansnotion** [comment]
Sure, the next version will support this feature.

**Comment by guigui0114** [comment]
**Feature Request**: Add independent keyboard shortcuts for different highlight colors.
**Current situation**: Only Ctrl+Shift+H is available, and it always uses the last color you picked.

**My usage**:
- Yellow (structure text): Ctrl+Shift+Y
- Green (key terms): Ctrl+Shift+G
- Red (top priority content): Ctrl+Shift+R

**Reason**:
Mouse clicking and color selection breaks my train of thought.
As someone with ADHD, I rely on color coding to quickly skim long text.
I need to mark key info instantly without interruption.


**需求**：希望为多种高亮颜色分别绑定独立快捷键。
**现状**：目前仅有 Ctrl+Shift+H 这一条高亮快捷键，只能固定沿用上次选中的颜色，无法一键切换色彩。

**自定义快捷键方案**：
举例，我在Obsidian里是这么设置的，我觉得批注起来比较丝滑
- 黄色（段落结构内容）：Ctrl+Shift+Y
- 绿色（重点语句词汇）：Ctrl+Shift+G
- 红色（最高优先级内容）：Ctrl+Shift+R
- 橘色（我自己的想法和批注，轻量，不用插入callout）：Ctrl+Shift+O

**诉求原因**：
- 反复点开颜色面板会打断思考节奏。我 ADHD，高度依赖色彩区分信息层级。
- 互联网上收集的内容、AI对话等上下文比较长。长文里的“废话”不删掉，二次阅读时效率很低；“废话”删掉可能影响理解。
- 高亮就能有效解决上面的痛点，实现跳读。色彩本身就能容纳信息，用不同色彩标注重点，跳读更快。
- 每个色彩都有对应的快捷键，批注的时候就会很丝滑。

**Comment by guigui0114** [comment]
**Feature Request**: Add format painter to copy and apply block styles quickly.

**Workflow I use now for low-level subtitles**:
1. Press Ctrl+Shift+0 to convert heading to plain text
2. Press ESC to select the whole block
3. Press Ctrl+B to bold the text

This multi-step operation is tedious. A format painter can copy finished block styles and apply them in one step.

**This is not an urgent requirement,** just a small tweak to make editing smoother.

**功能诉求**：新增格式刷功能，支持一键复制、复用完整区块样式。

**目前制作底层小标题需要三步操作**：

1. 快捷键 Ctrl+Shift+0 将标题转为纯文本
2. 按 ESC 选中整个区块
3. Ctrl+B 给文字加粗

整套操作步骤繁琐，如果有格式刷，直接复制已有成品区块样式就能一步完成。
**不算刚需紧急功能**，只是能大幅简化重复排版操作，提升编辑流畅度。

**Comment by joansnotion** [comment]
Great suggestion, I will implement it in the next version. Thank you very much!

**Comment by guigui0114** [comment]
大大，notion现在支持四级标题了，下个版本可以在大纲里显示一下四级标题~

**Comment by joansnotion** [comment]
> 大大，notion现在支持四级标题了，下个版本可以在大纲里显示一下四级标题~

It will be updated today, and is expected to be available tomorrow.

**Comment by joansnotion** [comment]
> **Feature Request**: Add independent keyboard shortcuts for different highlight colors. **Current situation**: Only Ctrl+Shift+H is available, and it always uses the last color you picked.
> 
> **My usage**:
> 
> * Yellow (structure text): Ctrl+Shift+Y
> * Green (key terms): Ctrl+Shift+G
> * Red (top priority content): Ctrl+Shift+R
> 
> **Reason**: Mouse clicking and color selection breaks my train of thought. As someone with ADHD, I rely on color coding to quickly skim long text. I need to mark key info instantly without interruption.
> 
> **需求**：希望为多种高亮颜色分别绑定独立快捷键。 **现状**：目前仅有 Ctrl+Shift+H 这一条高亮快捷键，只能固定沿用上次选中的颜色，无法一键切换色彩。
> 
> **自定义快捷键方案**： 举例，我在Obsidian里是这么设置的，我觉得批注起来比较丝滑
> 
> * 黄色（段落结构内容）：Ctrl+Shift+Y
> * 绿色（重点语句词汇）：Ctrl+Shift+G
> * 红色（最高优先级内容）：Ctrl+Shift+R
> * 橘色（我自己的想法和批注，轻量，不用插入callout）：Ctrl+Shift+O
> 
> **诉求原因**：
> 
> * 反复点开颜色面板会打断思考节奏。我 ADHD，高度依赖色彩区分信息层级。
> * 互联网上收集的内容、AI对话等上下文比较长。长文里的“废话”不删掉，二次阅读时效率很低；“废话”删掉可能影响理解。
> * 高亮就能有效解决上面的痛点，实现跳读。色彩本身就能容纳信息，用不同色彩标注重点，跳读更快。
> * 每个色彩都有对应的快捷键，批注的时候就会很丝滑。

Currently supports setting colors via shortcut keys

<img width="413" height="124" alt="Image" src="https://github.com/user-attachments/assets/0fd9e6b0-ba9b-4f7d-bb83-9aef47ed4bcb" />

<img width="406" height="593" alt="Image" src="https://github.com/user-attachments/assets/3b00847e-2644-4afa-a6ad-57421facc056" />
